platform/web/detect.py

```diff
@@ -112,6 +113,14 @@ def configure(env: "SConsEnvironment"):
         print_error("Initial memory must be a valid integer")
         sys.exit(255)
 
+    ## Add Emscripten to the included paths (for compile_commands.json completion)
```

Comment:
The `##` pattern was meant for section headers, like `## Build type`, `## Compiler configuration`, `## Dependencies` (see Linux `detect.py`).

This isn't really used consistently in the Web platform, but here it doesn't look correct.